README.md

```diff
@@ -140,6 +140,7 @@ The component accepts the following props:
 |     **`content`**     | function | A function that returns a component reference value. The content of this reference value is then used for print                     |
 |   **`copyStyles`**    | boolean  | Copies all &lt;style> and &lt;link type="stylesheet" /> from <head> inside the parent window into the print window. (default: true) |
 |  **`onBeforePrint`**  | function | Optional callback function that triggers before print. Either returns void or a Promise. If the function returns a Promise the content will be printed when the Promise is resolved. Users are responsible for catching the Promise rejecting.                                                                                     |
+| **`onBeforeGetContent`** | function | Optional callback function that triggers before lib get content to print. Either returns a Promise with a function. The content will be printed after the function returns. It's useful for manipulate the component to print before printing.
```

Comment:
Should be:

Optional callback function that triggers before the library gathers the page's content. Either returns void or a Promise. Users are responsible for catching the Promise rejecting. This can be used to change the content on the page before printing.

Reply:
changed